docs/fitting_types.rst

```diff
@@ -385,6 +385,28 @@ are added squared, ready to be minimized. Unlike in the above example, the
 
   Do not cite the overall :math:`R^2` given by :mod:`symfit`.
 
+Fitting moredimensional datasets
+--------------------------------
+Imagine that you have now a specific property over a grid, like a temperature,
+so you have a three-dimensional data set. To fit a function with two parameters
+to this data set, you can use :mod:`symfit`.
+
+At this example, we made a polynomial function with two coefficients,
```

Comment:
"At this example" should be "For this example". I think. I'm not a native english speaker.

Reply:
I checked it with grammarly. "At this example" should be correct.

Reply:
I'm definitely disagreeing with grammarly then ;). I would say 'in this example' or even better: 'As an example we made a polynomial function with two coefficients [...]'